### PR TITLE
MS-72: Clients crud

### PR DESCRIPTION
**Pull request by AlfonsoScornavacca**

### CODE REVIEW

**Review by AndresCespedes23 — APPROVED** [review]
Changes discussed via Discord

File: src/Components/Clients/Form/index.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    email: false,
+    address: false,
+    phoneNumber: false,
+    cuit: false,
+    activity: false
+  });
+
+  useEffect(() => {
+    if (id) {
+      fetch(`http://localhost:4000/api/clients/${id}`)
```
Comment (damianalvarez) [inline src/Components/Clients/Form/index.js:24]: @AlfonsoScornavacca let's use env variables instead of `http://localhost:4000`

**Review by damianalvarez — CHANGES_REQUESTED** [review]

File: src/Components/Clients/Form/form.module.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.formField {
```
Comment (damianalvarez) [inline src/Components/Clients/Form/form.module.css:1]: @AlfonsoScornavacca let's get rid of this file

**Review by damianalvarez — DISMISSED** [review]

File: src/Components/Shared/Modal/modal.module.css [hunk 3]
```diff
@@ -1,74 +1,74 @@
...
-    cursor: pointer;
-  }
-  
-  #cancelBtn {
-    background-color: crimson;
-  }
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(200, 200, 200);
+  position: sticky;
```
Comment (julianv97) [inline src/Components/Shared/Modal/modal.module.css:5]: this should be position:fixed;
so that you do not have the modal below the table.
image

**Review by julianv97 — CHANGES_REQUESTED** [review]

**Review by julianv97 — APPROVED** [review]

**Pull request merged**